common/src/main/java/apoc/ApocConfig.java

```diff
@@ -112,9 +117,18 @@ public class ApocConfig extends LifecycleAdapter {
 
     private List<IPAddressString> blockedIpRanges = List.of();
 
+    private boolean expandCommands;
+
+    private Duration commandEvaluationTimeout;
+
     public ApocConfig(Config neo4jConfig, LogService log, GlobalProcedures globalProceduresRegistry, DatabaseManagementService databaseManagementService) {
         this.neo4jConfig = neo4jConfig;
         this.blockedIpRanges = neo4jConfig.get(GraphDatabaseInternalSettings.cypher_ip_blocklist);
+        this.commandEvaluationTimeout = neo4jConfig.get(GraphDatabaseInternalSettings.config_command_evaluation_timeout);
+        if (this.commandEvaluationTimeout == null) {
+            this.commandEvaluationTimeout = GraphDatabaseInternalSettings.config_command_evaluation_timeout.defaultValue();
+        }
+        this.expandCommands = neo4jConfig.expandCommands();
```

Comment:
At this point, do we already know that config for expanding is set? Probably we shouldn't even try if the option is off.

Reply:
I'm not sure I follow, expand command can only be set on startup, so it would already be set as Neo4j starts APOC with the Neo4J config 🤔 Which part shouldn't we try?

Reply:
Sorry, my misunderstanding of the code, ignore this comment.